source/whats-new.txt

```diff
@@ -58,6 +58,24 @@ What's New in 6.0
      import these methods from the driver instead of from
      ``mongodb-client-encryption``.
 
+   - Removed support for strings with a length of 12 from the ``ObjectId``
+     constructor.
+
+   - The ``abortTransaction`` and ``commitTransaction`` methods return
+     ``null`` instead of the raw command results.
+
+   - Removed connection option helpers that recognized values other than ``true``
+     or  ``false`` as booleans. You must provide either ``true`` or ``false`` values for
+     options that expect boolean values in the connection string or MongoClient
+     constructor.
+
+   - Removed the Binary BSON type constructor that accepted a string.
+
+   - The ``Binary.write`` method no longer accepts a string to write
+     to the binary BSON object.
+
+   - ``ClientEncryption`` APIs return promises instead of callbacks.
```

Comment:
Suggestion:

I think this should be changed to be consistent: https://github.com/mongodb/docs-node/pull/741#discussion_r1296520856

Reply:
Oops, looks like I pushed without saving that change. Fixed